src/site/markdown/server/manage-webadmin.md

```diff
@@ -78,16 +105,16 @@ Response codes:
 Performs a health check for the given component. The component is referenced by its URL encoded name.
 
 ```
-curl -XGET http://ip:port/healthcheck/checks/Cassandra%20Backend
+curl -XGET http://ip:port/healthcheck/checks/Cassandra%20backend
 ```
 
 Will return the component's name, the component's escaped name, the health status and a cause.
 
 ```
 {
-  "componentName": "Cassandra Backend",
-  "escapedComponentName": "Cassandra%20Backend",
-  "status": "HEALTHY"
+  "componentName": "Cassandra backend",
+  "escapedComponentName": "Cassandra%20backend",
+  "status": "healthy"
```

Comment:
I'm not really fan of changing interfaces

Reply:
That's not changing interfaces that's changing inapropriate documentation!

Reply:
so, it was a mistake in the documentation? curl requests will given the same result?

Reply:
Bah yes of course!